### DISCUSSION TITLE
game crash when tp on server or switch dimensions

### DISCUSSION THREAD

**Discussion by KRIPOMC** [FabricMC/fabric-api#3039] [Player support]
A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x000002bf5cd1b540, pid=21540, tid=12880
#
# JRE version: OpenJDK Runtime Environment Microsoft-32931 (17.0.3+7) (build 17.0.3+7-LTS)
# Java VM: OpenJDK 64-Bit Server VM Microsoft-32931 (17.0.3+7-LTS, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# Problematic frame:
# C  [nvoglv64.dll+0x191b540]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# If you would like to submit a bug report, please visit:
#   https://github.com/microsoft/openjdk/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  S U M M A R Y ------------

Command Line: -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Dos.name=Windows 10 -Dos.version=10.0 -Xss1M -Djava.library.path=C:\Users\kisfo\AppData\Roaming\.minecraft\bin\8ba37763d54f03cb508134befcda07a749719caa -Dminecraft.launcher.brand=minecraft-launcher -Dminecraft.launcher.version=2.4.39 -DFabricMcEmu= net.minecraft.client.main.Main  -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -Dlog4j.configurationFile=C:\Users\kisfo\AppData\Roaming\.minecraft\assets\log_configs\client-1.12.xml net.fabricmc.loader.impl.launch.knot.KnotClient --username whisslewarrior --version fabric-loader-0.14.13-1.19.3 --gameDir C:\Users\kisfo\AppData\Roaming\.minecraft --assetsDir C:\Users\kisfo\AppData\Roaming\.minecraft\assets --assetIndex 2 --uuid 3d8a581bde4141c5b2f5be72534b3335 -ey6GK9pwiFho-a9VVmuh2YbWwdjtx8pfEsnBLc --clientId YTdjMTg3NTctOTRjZC00MzNiLWJmNjEtYTM1MWRiYjMwMmVm --xuid 2535419566252745 --userType msa --versionType release

Host: 11th Gen Intel(R) Core(TM) i9-11900KF @ 3.50GHz, 16 cores, 31G,  Windows 10 , 64 bit Build 19041 (10.0.19041.2788)
Time: Sat Apr 29 14:03:41 2023 Eastern Daylight Time elapsed time: 4386.427857 seconds (0d 1h 13m 6s)

---------------  T H R E A D  ---------------

Current thread (0x000002bf2cde0cf0):  JavaThread "Render thread" [_thread_in_native, id=12880, stack(0x000000db32500000,0x000000db32600000)]

Stack: [0x000000db32500000,0x000000db32600000],  sp=0x000000db325fea18,  free space=1018k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [nvoglv64.dll+0x191b540]

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 14419  org.lwjgl.opengl.GL11C.nglDrawElements(IIIJ)V (0 bytes) @ 0x000002bf38bcce6e [0x000002bf38bcce20+0x000000000000004e]
J 24200 c2 net.minecraft.class_286.method_43437(Lnet/minecraft/class_287$class_7433;)V (14 bytes) @ 0x000002bf38e0ff6c [0x000002bf38e0fd80+0x00000000000001ec]
J 24121 c2 net.minecraft.class_276.method_1233(IIZ)V (386 bytes) @ 0x000002bf390e68b0 [0x000002bf390e52e0+0x00000000000015d0]
J 26885 c2 net.minecraft.class_757.method_3192(FJZ)V (927 bytes) @ 0x000002bf3976abf0 [0x000002bf39769d80+0x0000000000000e70]
J 24885 c2 net.minecraft.class_310.method_1523(Z)V (1177 bytes) @ 0x000002bf394c8858 [0x000002bf394c7480+0x00000000000013d8]
J 27830% c2 net.minecraft.class_310.method_1514()V (276 bytes) @ 0x000002bf398e84a8 [0x000002bf398e8300+0x00000000000001a8]
j  net.minecraft.client.main.Main.method_44604([Ljava/lang/String;Z)V+1471
j  net.minecraft.client.main.Main.main([Ljava/lang/String;)V+2
j  java.lang.invoke.LambdaForm$DMH+0x00000001002ec000.invokeStaticInit(Ljava/lang/Object;Ljava/lang/Object;)V+10 java.base@17.0.3
j  java.lang.invoke.LambdaForm$MH+0x00000001002ec400.invokeExact_MT(Ljava/lang/Object;Ljava/lang/Object;Ljava/lang/Object;)V+19 java.base@17.0.3
j  net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(Ljava/lang/ClassLoader;)V+78
j  net.fabricmc.loader.impl.launch.knot.Knot.launch([Ljava/lang/String;Lnet/fabricmc/api/EnvType;)V+40
j  net.fabricmc.loader.impl.launch.knot.KnotClient.main([Ljava/lang/String;)V+4
v  ~StubRoutines::call_stub

siginfo: EXCEPTION_ACCESS_VIOLATION (0xc0000005), reading address 0x0000000000000000

**Comment by warjort** [comment]
`Problematic frame:
C [nvoglv64.dll+0x191b540]`

That's a crash in your graphics driver. Check it is up-to-date then contact nvidia.
